lib/Auth/Basic.php

```diff
@@ -430,7 +530,7 @@ public function verifyCredentials($user, $password)
                 } elseif (strlen($hash) == 40) {
                     $this->password_encryption = 'sha1';
                 } else {
-                    $this->password_encryption = false;
+                    $this->password_encryption = null;
```

Comment:
i'm not sure about this particular case but typically null and false have different meaning. `false` meaning that encryption is off. `null` might mean to use a best possible encryption. See "usePasswordEncryption()"